lib/opencensus/tags/formatters/binary.rb

```diff
@@ -48,12 +63,14 @@ class BinaryFormatterError < StandardError; end
         def serialize tags_context
           binary = [int_to_varint(VERSION_ID)]
 
-          tags_context.each do |key, value|
+          tags_context.each do |tag|
+            next unless tag.propagate?
+
             binary << int_to_varint(TAG_FIELD_ID)
-            binary << int_to_varint(key.length)
-            binary << key.encode(Encoding::UTF_8)
-            binary << int_to_varint(value ? value.length : 0)
-            binary << value.to_s.encode(Encoding::UTF_8)
+            binary << int_to_varint(tag.key.length)
+            binary << tag.key.encode(Encoding::UTF_8)
```

Comment:
Since tag keys and values may consist only of printable ASCII characters, I think we don't need this encoding.

Reply:
ok removing it.